kernel/sql-federation/optimizer/src/test/java/org/apache/shardingsphere/sqlfederation/optimizer/SQLOptimizeEngineTest.java

```diff
@@ -89,6 +89,11 @@ public void init() {
         Map<String, ShardingSphereTable> tables = new HashMap<>(2, 1);
         tables.put("t_order_federate", createOrderTableMetaData());
         tables.put("t_user_info", createUserInfoTableMetaData());
+        tables.put("t_order", createTOrderTableMetaData());
+        tables.put("t_order_item", createTOrderItemTableMetaData());
+        tables.put("t_single_table", createTSingleTableMetaData());
+        tables.put("t_order_federate_sharding", createTOrderFederateShardingMetaData());
```

Comment:
Can we use standard table? t_single_table, t_order_federate_sharding and t_order_item_federate_sharding will be removed in the future.

Reply:
You can refer issue #21286 for standard table.

Reply:
OK,I can update the case，After you finished the issue.